packages/prospectapi-common/src/lib.ts

```diff
@@ -221,3 +230,73 @@ export const deriveUrlMetadata = async (
     };
   }
 };
+
+/**
+ * use node fetch to hit the external metadata parser.
+ *
+ * broken out into a standalone function for easier mocking in tests.
+ *
+ * @param url
+ * @param metadataParserEnpointUrl
+ * @param metadataParserApiKey
+ * @returns JSON
+ */
+export const fetchUrlMetadata = async (
+  url: string,
+  metadataParserEnpointUrl: string,
+  metadataParserApiKey: string,
+): Promise<{ [key: string]: any } | null> => {
+  // Zyte-specific implementation
+  try {
+    const response = await fetch(metadataParserEnpointUrl, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Basic ${Buffer.from(
+          metadataParserApiKey + ':',
+        ).toString('base64')}`,
+      },
+      body: JSON.stringify({
+        url: url,
+        article: true,
+        articleOptions: {
+          extractFrom: 'httpResponseBody',
+        },
+        followRedirect: true,
+      }),
+    });
+
+    return await response.json();
+  } catch (e) {
+    Sentry.captureException(
+      new Error(`Metadata parser failed to return metadata for ${url}`),
+    );
+
+    return null;
+  }
+};
+
+export const convertMetadataJsonToClientApiItem = (
+  originalUrl: string,
+  metadataJson: {
+    [key: string]: any;
+  },
+): ClientApiItem => {
+  return {
+    ...(metadataJson.article && { excerpt: metadataJson.article.description }),
+    // TODO: does this two-character language match up with Pocket Parser behavior?
+    ...(metadataJson.article && { language: metadataJson.article.inLanguage }),
```

Comment:
I think we just need to uppercase this.

Zyte docs state it's `ISO 639-1` e.g. "en".

Our own language column contain these values:
```
EN
DE
FR
ES
IT
```